src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,46 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
-    public void getStatistic(String fromFileName, String toFileName) {
+    private static final int ONE = 1;
+    private static final int ZERO = 0;
+
+    public String getStatistic(String fromFileName, String toFileName) {
```

Comment:
The method `getStatistic` is doing multiple things: reading a file, processing data and writing to another file. According to the checklist, methods should be short and responsible for one task. Consider splitting this method into smaller methods with single responsibilities.

Reply:
fix comment